Makefile

```
@@ -240,6 +241,9 @@ CLI_ADMIN_JOBS_OCI_DISCOVERY := $(addprefix build-plugin-oci-,${ENVS})
 CLI_JOBS_LOCAL_DISCOVERY := $(addprefix build-cli-local-,${ENVS})
 CLI_ADMIN_JOBS_LOCAL_DISCOVERY := $(addprefix build-plugin-admin-local-,${ENVS})
 
+LOCAL_PUBLISH_PLUGINS_JOBS := $(addprefix publish-plugins-local-,${ENVS})
```

```suggestion
LOCAL_PUBLISH_PLUGINS_JOBS := $(addprefix publish-plugins-local-,$(ENVS))
```

lets use single notation for vars with parenthesis, we've a PR open streamlining this across the Makefile